handbook/engineering/hiring/software-engineer-frontend.md

```diff
@@ -46,18 +53,30 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
 ## Interview process
 
 1. You [apply here](https://jobs.lever.co/sourcegraph/b2f9a8b0-cc06-4629-81a0-0f2fa64271c7/apply).
-1. We set up a 30-minute call to learn more about what you are looking for, tell you about Sourcegraph, and answer any questions that you have.
-1. You complete a 3-hour [exercise project](software-engineer-coding-exercise.md#frontend-coding-exercise) that we designed to test the technical qualifications listed above.
-1. We review your submission and schedule a **1 hour follow-up call** to discuss it.
-1. We schedule additional interviews across multiple days.
-   - 1h **Technical experience:** We ask you about your past work and accomplishments in depth, how you worked with others, decisions you made, and what you'd do differently today.
-   - 1h **Team collaboration:** We ask you about how you work and communicate in a team setting, and how you handle tricky situations.
-   - 30m **VP Engineering**
-   - 30m **CTO**
-   - 30m **CEO**
+1. The **Hiring Manager** screens your application for suitability for the role by looking at:
+   - The answers you provide to the application questions.
+   - The skills you have obtained in your career.
+   - Your past work experience. In the absence of professional work experience we would look at any open-source projects you have created or contribute to.
+   - The [Nice-to-haves](#nice-to-haves) we listed above.
```

Comment:
nit (nonblocking): It feels like the JD itself is the definition of what we are looking for so it is redundant to elaborate here.

```suggestion
1. The **Hiring Manager** reviews all the information you provided on your application to determine if you meet our qualifications for this role (if there is another open role we think you would be better qualified for, we will let you know).
```